deployments/apiv2/services/users/src/main.py

```diff
@@ -96,35 +96,46 @@ async def login(request: Request, details: UserLogin | CustomerLogin):
         account_type = "customer"
         email = details.email.lower()
         select_query = (
-            "SELECT password, id, data->'scope' AS scope "
+            "SELECT password, id, data->'scope' AS scope, failed_login_attempts, suspended "
             "FROM customers WHERE deleted_at IS NULL AND email = $1"
         )
         select_query_params = (email,)
         customer_id = None
 
-        update_last_login_query = (
-            "UPDATE customers SET last_login = $1 WHERE deleted_at IS NULL AND email = $2"
-        )
+        update_last_login_query = "UPDATE customers SET last_login = $1, failed_login_attempts = 0 WHERE deleted_at IS NULL AND email = $2"
         update_last_login_params = (datetime.now(), email)
     else:
         account_type = "user"
         username = details.username.lower()
         # suspended is for deactivated accounts and verified is for new users needing to verify through email
         # select for service specific usage restrictions listed under the customer account
-        select_query = "SELECT password, id, data->'scope' AS scope FROM users WHERE deleted_at IS NULL AND name=$1 AND customer_id=$2 AND suspended='f' AND verified='t'"
+        select_query = (
+            "SELECT password, id, data->'scope' AS scope, failed_login_attempts, suspended FROM users "
+            "WHERE deleted_at IS NULL AND name=$1 AND customer_id=$2 AND verified='t'"
+        )
         select_query_params = (username, str(details.customer_id))
         customer_id = details.customer_id
 
-        update_last_login_query = "UPDATE users SET last_login = $1 WHERE deleted_at IS NULL AND name = $2 AND customer_id=$3 AND suspended='f' AND verified='t'"
+        update_last_login_query = "UPDATE users SET last_login = $1, failed_login_attempts = 0 WHERE deleted_at IS NULL AND name = $2 AND customer_id=$3 AND verified='t'"
```

Comment:
nitpick, can we keep the spacing around `=` consistent, this query has both `x=y` and `x = y`